spec/controllers/concerns/ordered_conferences_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+RSpec.describe OrderedConferences do
+  render_views
+
+  describe '#index' do
+    let!(:dummy_class) { Class.new { extend OrderedConferences } }
```

Comment:
The anonymous class you're defining doesn't have a `params` method, that's why you're getting the error. Further down you're checking for `@conferences` but that would reference an instance variable of the _test class_.

Does this code work? (only written here on GH, haven't run it anywhere)

```ruby
subject { dummy_class.new.index }

let!(:dummy_class) do
  Class.new do
    include OrderedConferences
    def params; {} end
    attr_reader :conferences # exposes the @conferences ivar
  end
end

# other let variables

it 'returns an ordered list of conferences' do
  expect(subject).to eq  [conference_africa, conference_europe, conference_na]
end
```